htdocs/public/payment/newpayment.php

```diff
@@ -882,6 +883,29 @@
 	print '<input type="hidden" name="tag" value="'.$tag.'">';
 	print '<input type="hidden" name="fulltag" value="'.$fulltag.'">';
 	print '</td></tr>'."\n";
+  
+if (! empty($conf->stripe->enabled) && (empty($conf->global->STRIPE_LIVE) || GETPOST('forcesandbox', 'alpha')) && empty($order->billed) && ($paymentmethod == 'stripe') ) {
+require_once DOL_DOCUMENT_ROOT.'/stripe/class/stripe.class.php';
+
+    $service = 'StripeTest';
+	$servicestatus = 0;
+ 
+	if (! empty($conf->global->STRIPE_LIVE))
```

Comment:
if GETPOST('forcesandbox', 'alpha') is on here, we will use the live when we want the sandbox.
Can you check this ?

Reply:
@eldy fixed !